covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -59,11 +84,12 @@ def _get_header_content(prediction_event: PredictionEvent, title: str):
         "story_fbid=10113020662000793&id=2247644"
     france_link = "https://www.reuters.com/article/us-health-coronavirus-france-toll/" \
         "french-coronavirus-cases-jump-above-chinas-after-including-nursing-home-tally-idUSKBN21L3BG"
+    next_day = prediction_event.date + timedelta(days=1)
     return [
         html.H1(children=title),
         html.P(children=[
             # TODO: fix the date
-            'On 2020-03-30, mathematicians Katarína Boďová and Richard Kollár ',
+            f'On {next_day}, mathematicians Katarína Boďová and Richard Kollár ',
```

Comment:
The fb link and number of countries are fixed. I'd suggest to rewrite to something like:
"B&K made predictions using a polynomial model. One prediction on 30.3 with 7 countries (fb link) and an update prediction on 13.4. with 99 countries (fb link). Below we show predictions until {prediction date}."

Reply:
I noticed line 121 also talks about dashed lines, we need to update it as well.

Reply:
There's plenty more that needs to be updated. I'd leave this to another PR, since this one is only about the dropdown button.